tests/runner.py

```diff
@@ -878,14 +878,20 @@ def assertIdentical(self, values, y, msg=None,
                                       fromfile=fromfile, tofile=tofile)
     diff = ''.join([a.rstrip() + '\n' for a in diff_lines])
     if EMTEST_VERBOSE:
-      print("Expected to have '%s' == '%s'" % limit_size(values[0]), limit_size(y))
+      print("Expected to have '%s' == '%s'" % (limit_size(values[0]), limit_size(y)))
     fail_message = 'Unexpected difference:\n' + limit_size(diff)
     if not EMTEST_VERBOSE:
       fail_message += '\nFor full output run with EMTEST_VERBOSE=1.'
     if msg:
       fail_message += '\n' + msg
     self.fail(fail_message)
 
+  def assertIdenticalUrlEncoded(self, expected, actual, **kwargs):
+    """
```

Comment:
Opening line of docstring does on the same line with the triple quote:  https://www.python.org/dev/peps/pep-0257/#id16